base/gfspvgmgr/virtual_group_manager.go

```diff
@@ -311,6 +318,9 @@ func (vgm *virtualGroupManager) refreshMetaByChain() {
 		log.Errorw("failed to list sps", "error", err)
 		return
 	}
+	sort.Slice(spList, func(i, j int) bool {
+		return spList[i].Id < spList[j].Id
+	})
```

Comment:
What is the meaning here?

Reply:
The `ListSP` API retrieved SP are shuffled, sort it in ascending order in order not to create duplicate GVG onchain